src/main/java/frc/robot/subsystems/TankDriveSubsystem.java

```diff
@@ -31,42 +32,33 @@ public class TankDriveSubsystem extends SubsystemBase {
     private Encoder quadratureEncoder1;
 
     public TankDriveSubsystem() {
```

Comment:
We should be able to collapse this class to just be
`this(false, false)`, call the 2-arg constructor with sensible values. This would reduce code duplication and possible bugs.